app/src/main/res/values/strings.xml

```diff
@@ -168,4 +168,6 @@
     <string name="verify_phone_number">ověřit telefonní číslo</string>
     <string name="battery_saver_settings_not_found">Vypněte si prosím spořič baterie ručně.</string>
     <string name="dashboard_not_verified_secondary">Ověřte prosím své telefonní číslo, abychom vás mohli kontaktovat v případě podezření na nákazu koronavirem.</string>
+    <string name="data_collection_info">Informace o sběru dat</string>
```

Comment:
All new strings should be uploaded to OneSky, translated to English and then downloaded back to this branch. Let me know your OneSky account, I will give you permissions.